src/options/processFileRenames.ts

```diff
@@ -41,36 +44,14 @@ const renameOptionFiles = async (options: TypeStatOptions, fileNames: ReadonlyAr
     };
 };
 
-const validRenameExtensions = new Set([".js", ".jsx"]);
+const validRenameExtensions = new Set([".cjs", ".cjsx", ".js", ".jsx", ".mjs", ".mjsx"]);
```

Comment:
...and, yes, these new friends.

Reply:
It's possible file renaming/grepping logic (#1030) doesn't yet account for them. I'll wait until someone files an issue.